doc/setup.md

```diff
@@ -93,8 +112,39 @@ These instructions assume that you have [Homebrew](http://brew.sh/) installed.
    conda activate skyportal_env
    conda install ligo.skymap
    ```
+<a name="configure-shell-mac"></a>
+### Configuring Shell Environment for Development
 
+When developing with SkyPortal on mac, you may  also need to configure your shell enviroment. Here is how you can do that by editing your `.zshrc` file.
 
+#### Open your `.zshrc` file in a text editor:
+
+```
+nano ~/.zshrc
+```
+#### Set enviroment variables to their installation paths 
+After installing the libraries with Homebrew, you'll need to set environment variables to their installation paths. Typically, Homebrew provides these paths upon successful installation. Replace the placeholder text `<path-to-library>` with the actual path that Homebrew provides upon sucessful installation.
+
+```
+export HDF5_DIR="<path-to-hdf5>"
+export BLOSC_DIR="<path-to-c-blosc>"
+export LZO_DIR="<path-to-lzo>"
+export BZIP_DIR="<path-to-bzip2>"
+```
+#### Alias pip3 and python3
+Depending on your system setup, the `python` and `pip` commands might point to Python 2 rather than Python 3. To ensure that you're using Python 3 and its corresponding pip version, you may need to set aliases in your `.zshrc` file:
+
+```
+alias pip='pip3'
+alias python='python3'
+```
+
+#### To activate the changes, source your .zshrc file:
+```
+source ~/.zshrc
+```
+### Checking for Port Availability 
```

Comment:
Maybe here mention what is the default port (5000), and that macports might be using it? Most folks won't care nor think about setting a custom port when starting the app for the first time